localization/sparse_mapping/src/vocab_tree.cc

```diff
@@ -65,10 +65,100 @@
 #pragma GCC diagnostic ignored "-Wdelete-non-virtual-dtor"
 #pragma GCC diagnostic push
 #include <DBoW2/DBoW2.h>      // BoW db that works with both float and binary descriptors
+#include <DBoW2/FSurf64.h>
 #pragma GCC diagnostic pop
 
 #include <vector>
 #include <string>
+#include <cmath>
+
+using namespace std;  // NOLINT (trying to modify this as little as possible)
+
+// stolen from file (this doesn't link for some reason?)
```

Comment:
Is there  a header/cc file you were trying to link from?

Reply:
This was from Brian's branch when he was working on the SURF vocab db. Not sure what it means.

Reply:
I think it's from DBoW2